circuit-lib/circuitlib-rs/src/verifying_keys/combined_26_1_1/mod.rs

```diff
@@ -0,0 +1,75 @@
+use groth16_solana::groth16::Groth16Verifyingkey;
+
+pub const VERIFYINGKEY: Groth16Verifyingkey = Groth16Verifyingkey {
+    nr_pubinputs: 5,
```

Comment:
The number of public inputs is one too high, can you change that in the script that generates these files?
(You can just decrease it by 1(vk_ic.length-1).)